lib/Perspective/Config.php

```diff
@@ -132,7 +132,7 @@ public static function getStandardPerspective(): array
                 'treeContextMenu' => [
                     'document' => [
                         'items' => [
-                            'addPrintPage' => \Pimcore\Config::getWeb2PrintConfig()['enableInDefaultView'] ? true : false, // hide add print documents by default
+                            'addPrintPage' => \Pimcore\Bundle\WebToPrintBundle\Config::getWeb2PrintConfig()['enableInDefaultView'] ? true : false, // hide add print documents by default
```

Comment:
Maybe we should refactor this in another way? Only add this when the bundle is enabled and installed?

Reply:
I removed this completely since it will be show if the bundle is installed.